{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/handlers.py

```diff
@@ -2,6 +2,10 @@
 from storages.backends.s3boto3 import S3Boto3Storage
 
 
+class StaticStorage(S3Boto3Storage):
+    location = settings.STATIC_FILES_LOCATION
```

Comment:
FYI: I don't think we have ever used S3 to host static files. On Heroku, we've always used Django Whitenoise.